config/jobs/flightdeck/flightdeck-periodics.yaml

```diff
@@ -0,0 +1,23 @@
+.selenium_small: &selenium_small
+  resources:
+    requests:
+      cpu: 2900m
+      memory: 2Gi
+
+periodics:
+
+- name: ci-flightdeck-staging-e2e
+  interval: 1h
+  spec:
+    containers:
+    - image: gcr.io/jetstack-workshops/portal-e2e:canary
```

Comment:
`eu.gcr.io/...`

Reply:
Also, instead of using `:canary`, you can use the sha256 sum of the image (i.e. `gcr.io/jetstack-workshops/portal-e2e@sha256:abcdefasudihasuda`)

Reply:
Should we be fixing it to a particular version, or just trying to use the latest build?